### ISSUE TITLE
 Class "\Keepa\objects\$string" not found

### ISSUE THREAD

**Issue by gokigoks** [keepacom/php_api#35]
We bumped our version from 2.0.5 to 2.0.6 and found the below error thrown from the serializer map() method. 

```
Class "\Keepa\objects\$string" not found {"exception":"[object] (Error(code: 0): Class \"\\Keepa\\objects\\$string\" not found at /home/forge/xxxreleases/20230317044233/vendor/netresearch/jsonmapper/src/JsonMapper.php:661)

#0 /home/forge/xxx/releases/20230317044233/vendor/netresearch/jsonmapper/src/JsonMapper.php(292): JsonMapper->createInstance()

#1 /home/forge/xxx/releases/20230317044233/vendor/netresearch/jsonmapper/src/JsonMapper.php(449): JsonMapper->map()

#2 /home/forge/xxx/releases/20230317044233/vendor/netresearch/jsonmapper/src/JsonMapper.php(282): JsonMapper->mapArray()

#3 /home/forge/xxx/releases/20230317044233/vendor/keepa/php_api/src/KeepaAPI.php(75): JsonMapper->map()

#4 /home/forge/xxx/releases/20230317044233/vendor/keepa/php_api/src/KeepaAPI.php(153): Keepa\\KeepaAPI->sendRequest()
```

We're using the sendRequestWithRetry().

**Comment by gokigoks** [comment]
The issue is happening on DE and JP locales as far as we checked. The issue is happening on version 2.0.6 downgrading to 2.0.5 fixed the issue. We checked on the difference between versions but didn't find anything that could have caused this. Please let us know if you get the chance to check.

**Comment by saschaarthur** [comment]
hello,

thanks for reporting the issue, can you provide us the code you using for doing the api calls?

**Comment by gokigoks** [comment]
Hi,

Here's a sample. 

```
$keepaRequest = new KeepaAPI(env('KEEPA_KEY'));
$request = Request::getProductRequest(
                    3,
                    0,
                    null,
                    null,
                    0,
                    1,
                    ['B07G3KVJT9'],
                    ['rating' => 1]
                );
$response = $keepaRequest->sendRequestWithRetry($request);
```

**Comment by saschaarthur** [comment]
Hello,

Sadly i was not able to reproduce the error with your code. But i guess i found the issue. Could you please check if its solved in 2.0.7.

**Comment by gokigoks** [comment]
Thank you @saschaarthur Will test locally and get back with an update
